### ISSUE TITLE
Configure missing secrets from workflow builder UI

### ISSUE THREAD

**Issue by zazzzSec** [TracecatHQ/tracecat#568]
**Is your feature request related to a problem? Please describe.**
Feature request. When configuring a Workflow with a previously unused integration, an error is thrown for the required secrets not being configured. Would be cool if the user is directly enabled to configured the required secrets right there.

**Describe the solution you'd like**
Instead of throwing an error for missing secrets, prompt the user to enter the required values.

**Describe alternatives you've considered**
none

**Additional context**
![image](https://github.com/user-attachments/assets/1713fe54-c448-4790-b248-5d78fa79428e)

**Comment by desaiparv5** [comment]
Faced similar issue. Found out that environment was set to `development` but I was initializing containers with `docker-compose.yaml` file.
Started the containers with `docker-compose.dev.yaml` and everything was working. :)

**Comment by topher-lo** [comment]
Makes sense! I think the "ideal" solution would be a dedicated integrations page with all the tools we support and a nice form for each. We will have more time in Q1 2025 to build out these UX improvements. If we're going to make this experience better, we want to make it excellent. 🙏
